foodsaving/utils/email_utils.py

```diff
@@ -18,11 +19,20 @@
 
 
 def date_filter(value):
-    return format_date(value, format='full', locale=to_locale(get_language()))
+    return format_date(
+        value.astimezone(get_current_timezone()),
```

Comment:
Seems weird that `format_date` doesn't support the `tzinfo` parameter. Is this equivalent?

Reply:
Yes, does seem odd. I think the logic was "oh, it's a date, not a time, so it doesn't need a timezone".

But _astimezone_ handles this fine. I think the function just doesn't care about the timezone, and asks for hour/minute/etc.

So, this is still a datetime with a timezone, but with the actual timezone, not utc. The point in time that it represents is still the same, just a different timezone perspective on it.